ontologies/works.ttl

```diff
@@ -0,0 +1,344 @@
+
+@prefix ww: <TBC/works> . 
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> . 
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix owl:     <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs:    <http://www.w3.org/2000/01/rdf-schema#> .
+
+TBC
+    rdf:type owl:Ontology ;
+    dcterms:title "Works"@en ;
+    dcterms:created "13/02/2017"^^xsd:date ;
+    dcterms:description "Ontology describing the properties of works"@en .
+	
+
+#######  Classes ##### 	
+
+ww:Work rdf:type owl:Class ;
+	rdfs:label "Work"@en ;
+	rdfs:comment "An individual work such as a text, archive item or picture; or a grouping of individual works (so, for instance, an archive collection counts as a work, as do all the series and individual files within it).  Each work may exist in multiple instances (e.g. copies of the same book).  N.B. this is not synonymous with "work" as that is understood in the International Federation of Library Associations and Institutions' Functional Requirements for Bibliographic Records model (FRBR) but represents something lower down the FRBR hierarchy, namely manifestation. Groups of related items are also included as works because they have similar properties to the individual ones."@en ;
+	rdf:isDefinedBy <TBC/works> .
+	
+ww:Collection rdf:type owl:Class ;
```

Comment:
Remove for now, pending further thinking over next couple of months. Think we might favour a more graph-like model where we just have Work with many relationship predicates.